src/meta/app/src/principal/user_info.rs

```diff
@@ -164,6 +164,8 @@ pub struct UserOption {
     network_policy: Option<String>,
 
     password_policy: Option<String>,
+
+    disabled: Option<bool>,
```

Comment:
How about changing this field to a key/value map? so that we can add other config properties in the future.